src/components/pages/packages/flyouts/deletePackage/index.js

```diff
@@ -1,4 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I think for all the renamed one's you can use git mv so it shows up as rename this will preserve history.

Reply:
Since these are just new flyouts in feature branch we don't need the history till now.